blocks/dialog/dialog.css

```diff
@@ -0,0 +1,31 @@
+.dialog dialog {
+  overscroll-behavior: none;
+  border: 1px solid #ccc;
+  border-radius: 16px;
+  left: 20px;
+  right: 20px;
+  width: clamp(300px, 80%, 700px);
+}
+
+.dialog dialog::backdrop {
+  background-color: rgb(0 0 0 / 50%);
+}
+
+.dialog .close-button {
```

Comment:
some of the style declarations here are (I think) no longer needed now that the content is an svg icon and not a textual `x`.

the other thought I have (and this may just be personal style) is that some of these sizes feel magic numbery to me (max-height, padding-right, font-size). If there is no better solution, no worries, just a thought.

Reply:
done